src/file-chooser.c

```diff
@@ -665,6 +680,28 @@ handle_save_file (XdpDbusFileChooser *object,
         g_variant_builder_add (&options, "{sv}", "current_file", g_variant_new_bytestring (host_path));
       }
   }
+  {
+      g_autoptr(GVariant) value =
+        g_variant_lookup_value (arg_options, "current_folder", G_VARIANT_TYPE_BYTESTRING);
+
+      if (value) {
```

Comment:
Niggle: move the { to the next line

Reply:
Apart from that, this looks good to me now.

Reply:
The indentation is wrong too